casanovo/casanovo.py

```
@@ -381,6 +400,31 @@ def configure(
     logger.info(f"Wrote {config_path}")
 
 
+def _is_valid_model(model: Optional[str], load_all_states: bool) -> None:
```

Ruff (0.15.6)

[warning] Boolean-typed positional argument in function definition

(FBT001)